test/Fondue.test.js

```diff
@@ -40,9 +40,7 @@ describe("The loaded font", () => {
 		const fondue = await loadFondue(
 			"./third_party/font.js/fonts/SourceCodeVariable-Roman.ttf"
 		);
-		expect(fondue.name("Copyright notice")).toContain(
-			"Adobe Systems Incorporated"
-		);
+		expect(fondue.name(1)).toContain("Source Code Variable");
```

Comment:
Is it no longer possible to retrieve from the name table by name? `fondue.name("Copyright notice")`.

Reply:
Indeed, this is not something that was needed anyway, see commit message in https://github.com/Wakamai-Fondue/wakamai-fondue-engine/pull/2/commits/7b05242e9567336c77dbfe5a6ae5b9a4d42e2a29